server/src/controllers/Chapter/inputs.ts

```diff
@@ -24,6 +24,9 @@ export class CreateChapterInputs implements Omit<Chapter, 'id' | 'creator_id'> {
 
   @Field(() => String)
   imageUrl: string;
+
+  @Field(() => String)
+  chatUrl: string;
```

Comment:
```suggestion
  @Field(() => String, { nullable: true })
  chatUrl?: string | null;
```
This is necessary if we make the field optional. Unfortunately, the fact it can now be null needs to be handled by `ChapterForm`.  For now it's fine to use

```ts
defaultValue={defaultValues[key] ?? undefined}
```

it's ugly, but it'll do until I come up with something nicer.